src/main/scala/nl.knaw.dans.easy.multideposit/parser/MultiDepositParser.scala

```diff
@@ -122,13 +123,14 @@ trait MultiDepositParser extends ParserUtils with InputPathExplorer
 
   def extractInstructions(depositId: DepositId, rows: DepositRows): Try[Instructions] = {
     val rowNum = rows.map(getRowNum).min
-
     checkValidChars(depositId, rowNum, "DATASET")
       .flatMap(dsId => Try { Instructions.curried }
         .map(_ (dsId))
         .map(_ (rowNum))
         .combine(extractNEL(rows, rowNum, "DEPOSITOR_ID").flatMap(exactlyOne(rowNum, List("DEPOSITOR_ID"))))
+        //.combine (atMostOne(rowNum, List("BASE_REVISION"))(extractList(rows, "BASE_REVISION").map(string => UUID.fromString(string))))
         .combine(extractProfile(rows, rowNum))
+        .combine(Try(atMostOne(rowNum, List("BASE_REVISION"))(extractList(rows, "BASE_REVISION")).get.get))
```

Comment:
we need to look at this line next week

1. The `.get`s don't seem right to me
2. `checkBaseRevisionConformsToUUID` needs to be incorporated here